etc/config/hylo.amazon.properties

```diff
@@ -0,0 +1,15 @@
+compilers=&hylo
+defaultCompiler=hylotrunk
+
+group.hylo.compilers=hylotrunk
+group.hylo.objdumper=/opt/compiler-explorer/gcc-11.1.0/bin/objdump
```

Comment:
Might as well use the latest gcc one at gcc-13.2.0 (not that it makes a huge difference :) )